### PR TITLE
Supporting tasks for MOSIP-11276

### PR DESCRIPTION
**Pull request by ase-101**

### CODE REVIEW

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/PartnerCACertEventSubscriber.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        logger.info("onApplicationEvent - scheduling subscription with startup delay: {}", initialSubscriptionDelay);
+        taskScheduler.schedule(
+                this::initSubsriptions,
+                new Date(System.currentTimeMillis() + initialSubscriptionDelay)
+        );
+        if (reSubscriptionDelaySecs > 0) {
+            logger.info("Work around for web-sub notification issue after some time.");
+            scheduleRetrySubscriptions();
+        } else {
+            logger.info("Scheduling for re-subscription is Disabled as the re-subsctription delay value is: "
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/PartnerCACertEventSubscriber.java:118]: can we avoid this concatenation?

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/Event.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package io.mosip.kernel.syncdata.websub.dto;
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/Event.java:1]: do we not have an existing class in kernel for this? as this is a generic format of the event?

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/EventModel.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package io.mosip.kernel.syncdata.websub.dto;
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/EventModel.java:1]: same as above can we not have it as part of kernel-core for all websub based events

File: admin/kernel-syncdata-service/src/test/resources/application.properties [hunk 4]
```diff
@@ -70,3 +59,63 @@ mosip.kernel.registrationclient.client.id=mosip-reg-client
...
+mosip.kernel.crypto.sign-algorithm-name=RS256
+#Certificate Sign algo name
+mosip.kernel.certificate.sign.algorithm=SHA256withRSA
+
+mosip.kernel.keymanager.softhsm.config-path=
+mosip.kernel.keymanager.softhsm.keystore-type=
+mosip.kernel.keymanager.softhsm.keystore-pass=
+
+mosip.kernel.keymanager.certificate.default.organizational-unit=MOSIP-TECH-CENTER
+mosip.kernel.keymanager.certificate.default.organization=IITB
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/test/resources/application.properties:107]: pls use mosip

File: admin/kernel-syncdata-service/src/test/resources/application.properties [hunk 5]
```diff
@@ -70,3 +59,63 @@ mosip.kernel.registrationclient.client.id=mosip-reg-client
...
+mosip.kernel.keymanager.certificate.default.location=BANGALORE
+mosip.kernel.keymanager.certificate.default.state=KA
+mosip.kernel.keymanager.certificate.default.country=IN
+
+mosip.kernel.keymanager.softhsm.certificate.common-name=www.mosip.io
+mosip.kernel.keymanager.softhsm.certificate.organizational-unit=MOSIP
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/test/resources/application.properties:113]: mosip-tech

File: admin/kernel-syncdata-service/src/test/resources/application.properties [hunk 6]
```diff
@@ -70,3 +59,63 @@ mosip.kernel.registrationclient.client.id=mosip-reg-client
...
+mosip.kernel.keymanager.softhsm.certificate.organization=IITB
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/test/resources/application.properties:114]: mosip

File: admin/kernel-syncdata-service/src/main/resources/application-local.properties [hunk 7]
```diff
@@ -90,4 +69,64 @@ mosip.kernel.pdf_owner_password=test
...
+
+mosip.kernel.keymanager.certificate.default.organizational-unit=MOSIP-TECH-CENTER
+mosip.kernel.keymanager.certificate.default.organization=IITB
+mosip.kernel.keymanager.certificate.default.location=BANGALORE
+mosip.kernel.keymanager.certificate.default.state=KA
+mosip.kernel.keymanager.certificate.default.country=IN
+
+mosip.kernel.keymanager.softhsm.certificate.common-name=www.mosip.io
+mosip.kernel.keymanager.softhsm.certificate.organizational-unit=MOSIP
+mosip.kernel.keymanager.softhsm.certificate.organization=IITB
```
Comment (gsasikumar) [inline admin/kernel-syncdata-service/src/main/resources/application-local.properties:126]: org as mosip and ou as mosip-tech

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/EventModel.java (hunk 3 above)
Comment (ase-101) [reply admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/EventModel.java]: moved models to kernel-core

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**